packages/orchestration/src/exos/chain-hub.js

```diff
@@ -41,6 +44,20 @@ import { getBech32Prefix } from '../utils/address.js';
  * @see {ChainHub} `registerAsset` method
  */
 
+/**
+ * Object used to help build a MsgTransfer (ibc transfer).
+ *
+ * If it contains `forwardInfo`, this must be stringified and provided as the
+ * `memo` for `MsgTransfer`.
+ *
+ * @typedef {object} TransferRoute
+ * @property {string} sourcePort
+ * @property {IBCChannelID} sourceChannel
+ * @property {Coin} token
+ * @property {string} receiver - destination chain address value
```

Comment:
In TS I'm not sure how to express this, maybe @turadg has ideas. Runtime checks are certainly an option but we can also rely on the cosmos layer to surface bech32 errors.

Here, `receiver` is not required to be valid `bech32`. If we have `forwardInfo` this will be `"pfm"`. _Supposes this part could be expressed in the types._